lib/matplotlib/__init__.py

```diff
@@ -909,8 +911,17 @@ def __setitem__(self, key, val):
                 '%s is not a valid rc parameter. See rcParams.keys() for a '
                 'list of valid parameters.' % (key,))
 
+    def _set_current_backend(self, backend):
+        dict.__setitem__(self, 'backend', backend)
+
     def __getitem__(self, key):
-        if key in _deprecated_map:
+        if key == 'backend':
+            cbook.warn_deprecated('3.0',
+                "The rcParam 'backend' is deprecated. Use the rcParam "
+                "'default_backends' to access the defaults. Use get_backend() "
+                "to get the current backend.")
```

Comment:
But then you need to store the current backend somewhere else and update the implementation of get_backend accordingly.